### PR TITLE
feat: notifications history ui

### PR DESCRIPTION
**Pull request by ice-hades**
Recording:

https://github.com/user-attachments/assets/b93357ac-31e3-4e2d-988f-0a307fa61df5

### CODE REVIEW

File: lib/app/features/feed/views/pages/notifications_history_page/components/notification_item/notification_item.dart [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  static double get padding => ScreenSideOffset.defaultSmallMargin;
+
+  static double get separator => 4.0.s;
+
+  static int get iconsCount => 10;
+
+  @override
+  Widget build(BuildContext context) {
+    final iconSize =
+        ((MediaQuery.of(context).size.width - padding * 2) - separator * (iconsCount - 1)) /
```
Comment (ice-kreios) [inline lib/app/features/feed/views/pages/notifications_history_page/components/notification_item/notification_item.dart:29]: could we use `MediaQuery.sizeOf(context).width`?

> By calling MediaQuery.of(context).size/padding/orientation, the widget will rebuild when any of the MediaQuery properties change (less performant). By calling MediaQuery.sizeOf/paddingOf/orientationOf(context) the widget will rebuild only when the property changes, avoiding unnecessary rebuilds.

**Review by ice-kreios — DISMISSED** [review]

File: lib/l10n/app_en.arb [hunk 2]
```diff
@@ -2,6 +2,8 @@
   "@@locale": "en",
   "day": "{count, plural, =1{day} other{days}}",
   "hour": "{count, plural, =1{hour} other{hours}}",
+  "h": "{hours}h",
```
Comment (ice-tychon) [inline lib/l10n/app_en.arb:5]: `h` is not a descriptive name. Please, change it.

File: lib/app/features/feed/views/pages/notifications_history_page/components/notification_item/notification_info.dart [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+          color: Colors.white,
+          child: SizedBox(
+            width: 240.0.s,
+            height: 20.0.s,
+          ),
+        ),
+      );
+    }
+
+    final newTapRecognizers = <TapGestureRecognizer>[];
```
Comment (ice-tychon) [inline lib/app/features/feed/views/pages/notifications_history_page/components/notification_item/notification_info.dart:51]: Please, extract his section to a separate function.

File: lib/l10n/app_en.arb (hunk 2 above)
Comment (ice-tychon) [reply lib/l10n/app_en.arb:5]: This same goes for `d`

**Review by ice-kreios — APPROVED** [review]

**Review by ice-alcides — APPROVED** [review]

**Pull request merged**